src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { ItemResponseType } from 'shared/consts';
+
+export const DisplayModeOptions = {
+  [ItemResponseType.MultipleSelection]: [
+    { id: 'sentence', name: 'Sentence' },
+    { id: 'bullet_list', name: 'Bullet List' },
+  ],
+  [ItemResponseType.SingleSelectionPerRow]: [
+    { id: 'sentence_option_row', name: 'Sentence: Option Text, Row Text' },
+    { id: 'sentence_row_option', name: 'Sentence: Row Text, Option Text' },
+    { id: 'bullet_list_option_row', name: 'Bullet List: Option Text, Row Text' },
+    { id: 'bullet_list_text_row', name: 'Bullet List: Row Text, Option Text' },
+  ],
+  [ItemResponseType.MultipleSelectionPerRow]: [
+    { id: 'sentence_option_row', name: 'Sentence: Option Text, Row Text' },
+    { id: 'sentence_row_option', name: 'Sentence: Row Text, Option Text' },
+    { id: 'bullet_list_option_row', name: 'Bullet List: Option Text, Row Text' },
+    { id: 'bullet_list_text_row', name: 'Bullet List: Row Text, Option Text' },
+  ],
+  [ItemResponseType.Slider]: [
+    { id: 'slider_one', name: 'Slider 1' },
+    { id: 'slider_two', name: 'Slider 2' },
+  ],
```

Comment:
I'm sure this was just a slip-up, but the `Slider` type doesn't use the "Display Mode" dropdown, according to the AC:

![CleanShot 2024-08-08 at 15 54 43@2x](https://github.com/user-attachments/assets/b1b5c1f7-2768-4010-9467-c09df3c61201)

```suggestion
  [ItemResponseType.Slider]: undefined,
```

Perhaps you were thinking `SliderRows` here? I assume `SliderRows` support for the distinct "Response From" dropdown will be getting added in a separate PR that you'll create for [M2-7211](https://mindlogger.atlassian.net/browse/M2-7211) – is that right?